build.sbt

```diff
@@ -31,13 +30,13 @@ lazy val root = (project in file("."))
       Cats.core,
       Cats.effect,
       Cats.log4cats,
-      Cats.log4catsSlf4j,
+      Cats.log4catsSlf4j  % Test,
```

Comment:
This is only used for testing to actually print Logs. In the real world the user will provide the logging backend, we shouldn't assume Slf4j.